ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -1,23 +1,322 @@
+      </header>
+
+      <div>
+        <section class="pa-6">
+          <v-row no-gutters class="">
+            <v-col cols="3">
+              <h3>Unit Note Type</h3>
+            </v-col>
+            <v-col cols="9">
+              {{ unitNoteType.header }}
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col cols="3">
+              <h3>Document ID</h3>
+            </v-col>
+            <v-col cols="9">
+              {{ getMhrUnitNote.documentId }}
+            </v-col>
+          </v-row>
+          <v-divider class="my-3 mx-0" />
+          <v-row no-gutters>
+            <v-col cols="3">
+              <h3>Remarks</h3>
+            </v-col>
+            <v-col cols="9">
+              <p>{{ getMhrUnitNote.remarks }}</p>
+            </v-col>
+          </v-row>
+          <v-divider class="my-3 mx-0" />
+          <div class="px">
+            <h3>Person Giving Notice</h3>
+
+            <v-simple-table v-if="getMhrUnitNote" class="giving-notice-party-table" data-test-id="account-info-table">
+              <template v-slot:default>
+                <thead>
+                  <tr>
+                    <th class="px-0">
+                      Name
+                    </th>
+                    <th class="px-0">
+                      Mailing Address
+                    </th>
+                    <th class="px-0">
+                      Email Address
+                    </th>
+                    <th class="px-0">
+                      Phone Number
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="person-name">
+                      <v-icon class="mt-n2">
+                        {{ getMhrUnitNote.businessName ? 'mdi-domain' : 'mdi-account' }}
+                      </v-icon>
+                      <span class="font-weight-bold">
+                        {{ givingNoticeParty.personName.first }}
+                        {{ givingNoticeParty.personName.middle }}
+                        {{ givingNoticeParty.personName.last }}
+                      </span>
+                    </td>
+                    <td class="">
+                      <base-address
+                        :editing="false"
+                        :schema="PartyAddressSchema"
+                        :value="givingNoticeParty.address"
+                      />
+                    </td>
+                    <td class="">
+                      {{ givingNoticeParty.emailAddress }}
+                    </td>
+                    <td class="">
+                      {{ toDisplayPhone(givingNoticeParty.phoneNumber) }}
+                      <span v-if="givingNoticeParty.phoneExtension">
+                        Ext {{ givingNoticeParty.phoneExtension }}
+                      </span>
+                    </td>
+                  </tr>
+                </tbody>
+              </template>
+            </v-simple-table>
```

Comment:
Looks good! 
Is this simple enough to be a basic reusable component?

Reply:
I asked Liz to provide additional design samples, and after all variations are established, I can put into own component.

Reply:
Created own component for the Unit Notes details.

Reply:
Cool!